docs/community/developer-guide.md

```diff
@@ -134,3 +134,18 @@ Example launch.json file:
   ]
 }
 ```
+
+## Test pipeline
+
+AKS-Engine employs a Continuous Integration (CI) system that incorporates Azure DevOps, configured to interact with the AKS-Engine GitHub project.
+
+The following steps constitute the AKS-Engine CI pipeline:
+
+1. Contributor opens a Pull Request (PR) against the AKS-Engine project
+1. The PR triggers an Azure DevOps job that
+   + applies the changes to the HEAD of the master branch
+   + generates multiple ARM templates for different deployment scenarios
+   + simultaneously provisions the clusters based on generated templates in Azure
+1. The PR is code reviewed by the members of AKS-Engine team
+1. Once the PR is approved and the end-to-end job has passed, the PR can now be merged into the master branch
```

Comment:
should we mention unit test coverage here too? since it's also a requirement

Reply:
Yeah sounds good. FWIW this was largely a lift n' shift from the [AKS Engine code delivery guide](https://github.com/Azure/aks-engine/blob/master/docs/aksengine-code-delivery.md) so it probably had quite a few gaps since it mentioned Jenkins, DC/OS, swarm etc. I can definitely add those in though as part of the lift n' shift

Reply:
Perfect thank you! I hadn't realized that this was an existing list, it was a good call to refresh it since it was probably quite outdated.